lib/diagnostic.rb

```diff
@@ -31,8 +31,9 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = Hash.new { |hash, key| hash[key] = '#{key} is really a Sith Lord.'
 
+Diagnostic.interpolation[:character]
```

Comment:
We can interpolate variables into strings using `#{}` with Ruby
So to interpolate our variable `character` we could do 
`"#{character} is really a Sith Lord."`